app_server/app/Router.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Router module. The main entry point for worldscope application server
+ * @module Router
+ */
+var Hapi = require('hapi');
+var Utility = require('local/Utility');
+
+var logger = Utility.createLogger(__filename);
+
+/* Configure Hapi server connection */
+var server = new Hapi.Server();
+server.connection({ port: 3000 });
+server.route({
+  method: 'GET',
+  path: '/',
+  handler: function (request, reply) {
+    reply("Welcome to WorldScope");
```

Comment:
`'` instead of `"`?
Likewise for the rest of this file and `test/local/Utility.js`.